feedthefox/devices/admin.py

```diff
@@ -3,14 +3,13 @@
 from feedthefox.devices.models import Build, Device, DeviceInfo
 
 
-class DeviceInfoInline(admin.TabularInline):
-    model = DeviceInfo
-    extra = 1
+@admin.register(DeviceInfo)
+class DeviceInfoAdmin(admin.ModelAdmin):
+    list_display = ('user', 'device', 'imei',)
```

Comment:
``user`` string representation is the full name. I suggest either change list display to ``mozillians_username`` or change the string representation (if there aren't any references to it). Full name is not always available.